### PR TITLE
build: Additions for power platforms

### PR DESCRIPTION
**Pull request by mhdawson**
I'm not sure if its too early in the process yet for the additions under the ### Configurations: Core & difficult
section but our goal is to get them into the Core set when possible so this seemed like the right place for now given the description.  Let me know if I'm wrong on that front

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,9 @@ The following platforms will eventually be included in the "Core" set when prope
   * ARMv8 64-bit (Linux, one day, when suitable hardware & OS is available)
   * Ubuntu 14.10 (Utopic Unicorn) 64-bit (preferably prior to official release)
   * Mac OS X 10.10 (Yosemite) + XCode 6
```
Comment (Fishrock123) [inline README.md]: I feel like the above lists probably need some updating too haha

Comment (mhdawson) [inline README.md]: Agreed it seemed a bit stale in some places.  @jbergstroem if there are other updates I should pull into this PR just let me know

Comment (jbergstroem) [inline README.md]: There was changes to this list as part of another PR somewhere if I recall correctly. Lets just update it separately?

**Comment by mhdawson** [comment]
@jbergstroem could I get your ok and then I'll go ahead and land this.

**Comment by jbergstroem** [comment]
LGTM, sorry for the delay.

**Comment by mhdawson** [comment]
Thanks, I'll land when I'm back in the office next week

**Comment by mhdawson** [comment]
landed as https://github.com/nodejs/build/commit/15e8935e3f2b154084e5006ac0eecfa77a9b44fb

**Pull request closed without merging**